manifests/base/grafana-v5/credentials.yaml

```diff
@@ -1,8 +0,0 @@
-kind: Secret
```

Comment:
cant we keep those and the influx-grafana-auth as a default?
For a development environment where we don't want sealed secrets, or a kind cluster.

IIRC I kept those, because I cant remove resources in overlays. 
But it might be possible according to
https://github.com/kubernetes-sigs/kustomize/issues/1593#issuecomment-1190801833